Config/DefaultEngineBase.ini

```diff
@@ -2,8 +2,9 @@
 
 [/Script/EngineSettings.GameMapsSettings]
 GameDefaultMap=/Game/Maps/${LEVEL_NAME}.${LEVEL_NAME}
+ServerDefaultMap=/Game/Maps/${LEVEL_NAME}.${LEVEL_NAME}
 EditorStartupMap=/Game/Maps/${LEVEL_NAME}.${LEVEL_NAME}
-GlobalDefaultGameMode=/Script/RapyutaSimulationPlugins.RRROS2GameMode
+GlobalDefaultGameMode=/Script/RapyutaSimulationPlugins.RRNetworkGameMode
```

Comment:
I don't think its too necessary for the Editor, additionally when we try running with PIE, this doesnt work since the command line is empty when PIE is run. With the Editor its easy to just change the map and gamemode, so its probably not too necessary to change the Base.ini, so I think we can just set the Base.ini to the entry map and let the user change the maps and game modes through the editor's own tools. Also they can still emulate the full behavior of the entry map in the packaged client by using play standalone